swift/actions/build-and-test/action.yml

```diff
@@ -44,10 +44,10 @@ runs:
         mkdir -p bazel-cache/{repository,disk}
         echo build --repository_cache=bazel-cache/repository --disk_cache=bazel-cache/disk > local.bazelrc
         echo test --test_output=errors >> local.bazelrc
-    - name: Print unextracted entities
-      shell: bash
-      run: |
-        bazel run //swift/extractor/print_unextracted
+    # - name: Print unextracted entities
```

Comment:
Is it necessary to comment this step?

Reply:
We could probably remove it, but I think we should actually fix it (the tracking issue is here https://github.com/github/codeql-c-team/issues/1780) in a followup PR.